BeatTogether.DedicatedServer.Kernel/Managers/GameplayManager.cs

```diff
@@ -23,9 +22,9 @@ public sealed class GameplayManager : IGameplayManager, IDisposable
         public GameplayModifiers? CurrentModifiers { get; private set; }
 
         private const float SongStartDelay = 0.5f;
-        private const float ResultsScreenTime = 30f;
-        private const float SceneLoadTimeLimit = 15.0f;
-        private const float SongLoadTimeLimit = 15.0f;
+        private const float ResultsScreenTime = 20f; //changing this to 20 sec as on quest i think it is that
+        private const float SceneLoadTimeLimit = 10.0f;
+        private const float SongLoadTimeLimit = 10.0f;

```

Comment:
shortened loading time limits, quest takes 20 seconds then is returned to the lobby so i set it to 20 so the start / ready buttons would not stay greyed out for 10 sec after they returned to the lobby.